oak/proto/log.proto

```diff
@@ -42,6 +42,6 @@ enum Level {
   ERROR = 1;
   WARN = 2;
   INFO = 3;
-  DEBUG = 4;
+  DEBUGGING = 4;
```

Comment:
Why the change?

Reply:
"This avoids clashes with C++ builds that use DEBUG as a preprocessor macro." (from the commit message)

Reply:
I can't really see the comments while reviewing, unless I review one by on. Am I just an idiot and didn't notice where they are displayed (when looking at all the changes for the PR)?